real-examples/tools/convert_releases_to_bigquery_schema.py

```diff
@@ -528,7 +528,7 @@ def main():
 
     all_data = []
     files = glob.glob('%s*' % options.filepath)
-    schema = json.load(open('release-schema.json'))
+    schema = json.load(open('release-schema.json', 'r'))
```

Comment:
I added 'r' as to fail fast in case release-schema is not available, for consistency with the other script, and to explicitly making it read-only in case it exists

Reply:
In Python 3 the default is 'r' so it's not necessary to specify: https://docs.python.org/3.6/library/functions.html#open

`open` will fail fast regardless, and if it succeeds it will still be read-only.